content/docs/instrumenting/exporters.md

```diff
@@ -138,6 +138,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [NRPE exporter](https://github.com/robustperception/nrpe_exporter)
    * [Osquery exporter](https://github.com/zwopir/osquery_exporter)
    * [OTC CloudEye exporter](https://github.com/tiagoReichert/otc-cloudeye-prometheus-exporter)
+   * [Smokeping prober](https://github.com/SuperQ/smokeping_prober)
```

Comment:
This is really misc, as you're not talking to an existing smokeping setup

Reply:
Agreed; else LGTM.

Reply:
so the reason i put it in "other monitoring system" in the first place is that the original solution i proposed was using "fping" (the "other") instead of a home-grown implementation. but I guess that is, indeed, not a valid argument anymore.

i'm concerned about adding yet another entry to "misc". there's already way too much disparate stuff in there, it's hard to make head or tails in there...

Reply:
That's what tends to happen with any Misc category in any system.

Reply:
heh. i guess what i'm saying is we might be able to create a new category from the misc one... maybe we could regroup the following in a "Network service" section of some sort?

* BIND
* Blackbox
* Dovecot
* PowerDNS
* rTorrent
* SMTP...
* Transmission
* Unbound

Then the ICMP/Ping probers could fit right in.

Reply:
Why would Dovecot be a network service and BIG-IP not? Almost all of the misc list are for services provided over the network.

I'd suggest getting this in, and then if you've a better idea for taxonomy exploring that in another issue/PR.

Reply:
because i don't know what BIG-IP is. :) My argument is that there *are* a bunch of things in misc that are not *primarily* network services. cAdvisor, JIRA, Jenkins, kube-state... those are of course available over the network, but their primary purpose is not to offer some standard network protocol (like DNS or IMAP) but to provide a service (like bugtracking or CI).

but you are right: maybe that peculiar bikeshed should be painted somewhere else.

Reply:
BIG-IP is a hardware http load-balancer appliance product/company.

Reply:
Actually BIG-IP should be under hardware.